AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/capture/DefaultCameraCaptureSource.swift

```diff
@@ -225,8 +225,8 @@ import UIKit
         let newAVFormat = captureDevice.formats.min { avFormatA, avFormatB in
             let formatA = VideoCaptureFormat.fromAVCaptureDeviceFormat(format: avFormatA)
             let formatB = VideoCaptureFormat.fromAVCaptureDeviceFormat(format: avFormatB)
-            let diffA = abs(formatA.width - format.width) + abs(formatA.height - format.height)
-            let diffB = abs(formatB.width - format.width) + abs(formatB.height - format.height)
+            let diffA = abs(formatA.width - format.width) + abs(formatA.height - format.height) + abs(formatA.maxFrameRate - format.maxFrameRate)
+            let diffB = abs(formatB.width - format.width) + abs(formatB.height - format.height) + abs(formatB.maxFrameRate - format.maxFrameRate)
```

Comment:
This creates an arbitrary preference of matching resolution over matching frame rate (not that that isn't probably reasonable for most use cases) simply given that the values of resolution (and therefore the differences) are larger. This isn't a big deal because builders should really just be enumerating and providing exact matches, but we should still either:
* Normalize the differences and apply some weights.
* (Preferred) Just match resolution first, and then match framerate after. Which basically gives 100% preference to resolution but additionally is more explicit and easy to reason above.

Reply:
updated to filter by fps and then match resolution.

Reply:
updated to match fps after resolution.